### ISSUE TITLE
Can't watch nothing Error : Stream is not Supported

### ISSUE THREAD

**Issue by BTr-RJ** [Misunderstood-Wookiee/Stremio-UWP#1]
If i try to watch the content directly i get this "Erros: Stream is not Supported" message.
If i try to paste the magnet link, nothing happens.

How is this supposed to work?

**Comment by BTr-RJ** [comment]
Ok, i managed to make it work after allowing remote connections from my PC (don't expected to work like this), but content is losing a portion of the screen and it's not using any Xbox video resource like HDR or Dolby Vision. Is there any plans to Stremio be totally standalone and using Xbox resources? It looks very rough to me as it is now

**Comment by Misunderstood-Wookiee** [comment]
> Ok, i managed to make it work after allowing remote connections from my PC (don't expected to work like this), but content is losing a portion of the screen and it's not using any Xbox video resource like HDR or Dolby Vision. Is there any plans to Stremio be totally standalone and using Xbox resources? It looks very rough to me as it is now

Yes this was a proof of concept inititally I did have plans to further develop it and try and overcome such issues.
